content/docs/user-guide/dvc.md

```diff
@@ -147,3 +147,21 @@ metafile (`models/rf.mlem`) and `dvc.lock` only.
 
 Learn more about [DVC](https://dvc.org/doc) and how it can be useful for
 training your ML models.
+
+## Working with private repositories
+
+If you commit model metafiles to a private repo and use DVC to store binaries,
+you'll need to authorize both via SSH (for DVC) and via HTTPS (for MLEM).
+
+SSH authorization is usually achieved by running `git push` against a SSH
+remote, or it can be done with
+[`gh auth login`](https://cli.github.com/manual/gh_auth_login) (if you use
+Github).
+
+HTTPS authorization is done by setting `GITHUB_USERNAME` and `GITHUB_TOKEN` env
+vars. You can generate a token [here](https://github.com/settings/tokens).
```

Comment:
env vars -> environment variabls
Also it's not clear here if you need or not to generate a token ("can" ?)

Reply:
Yep, it's a limitation. I'll add few sentences.

Reply:
I don't really know why DVC doesn't work with HTTP authentication. Maybe @skshetry can briefly explain the reason.